src/aiidalab_qe/app/configuration/basic/basic.py

```diff
@@ -50,9 +54,18 @@ def render(self):
             "value",
         )
 
+        self.starting_magmom_warning = ipw.HTML(
+            value="""
+                <div class="alert alert-inline alert-info" style="margin-left: 10px;">
+                    Can only set total magnetization for insulating systems
```

Comment:
Note: only total magnetization can be set for insulators
```suggestion
                    Can only set total magnetization for insulating systems
```

Reply:
Yeah, I wasn't all too happy with my phrasing. Okay, in that case, I will make messages in the format of "Note: ..." 👍